cmd/resources/resources.go

```diff
@@ -29,6 +29,7 @@ type TemplateData struct {
 
 type ResourceData struct {
 	Name        string
+	DisplayName string
```

Comment:
I can also pass in functions to the template to handle converting these values from camel/kebab, which I might do in a refactor